version/1/2/0/code_of_conduct.md

```diff
@@ -0,0 +1,23 @@
+# Contributor Code of Conduct
+
+As contributors and maintainers of this project, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
+
+We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or political affiliation.
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexual language or imagery
+* Derogatory comments
+* Personal attacks
+* Trolling or insulting comments
+* Public or private harassment
+* Publishing other's private information, such as physical or electronic addresses, without permission
+* Other unethical or unprofessional conduct.
```

Comment:
> Well, yeah, project admins get flexibility for interpretation and enforcement against otherwise un-enumerated behaviors.

Isn't this the problem we're trying to solve? Historically, maintainers have not been good at identifying and punishing harassment; often even defending the harasser. Faux Codes of Conduct (ex: "be excellent to each other") have not worked due to flexible interpretation. By having an explicit Code of Conduct it removes any doubt or question about what's isn't acceptable and what course of action needs to be taken.

> Otherwise you're giving the "everything not explicitly forbidden is allowed" crowd free rein by tying the admins' hands

By leaving the Code of Conduct open ended we're also leaving a loophole for those that claim they are being persecuted due to "reverse sexism". Also, I'm pretty sure your hypothetical example violates Twitter's ToS, as it crosses the line between parody and defamation/libel.